zip-0317.rst

```diff
@@ -8,9 +8,308 @@
            Daira Hopwood
            Jack Grigg
            Francisco Gindre
-  Status: Reserved
+  Status: Draft
   Category: Standards / Wallet
+  Obsoletes: ZIP 313
   Created: 2022-08-15
   License: MIT
   Discussions-To: <https://forum.zcashcommunity.com/t/zip-proportional-output-fee-mechanism-pofm/42808>
   Pull-Request: <https://github.com/zcash/zips/pull/631>
+
+
+Terminology
+===========
+
+The key word "SHOULD" in this document is to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "conventional transaction fee" in this document is in reference
+to the value of a transaction fee that is conventionally used by wallets,
+and that a user can reasonably expect miners on the Zcash network to accept
+for including a transaction in a block.
+
+The terms "Mainnet, "Testnet", and "zatoshi" in this document are defined
+as in [#protocol-networks]_.
+
+
+Abstract
+========
+
+The goal of this ZIP is to change the conventional fees for transactions
+and get buy-in from wallet developers, miners and Zcash users.
+
+
+Motivation
+==========
+
+In light of recent network activity, it is time to review and update the
+standard 1,000 zatoshi transaction fee set in ZIP 313 [#zip-0313]_.
+
+The conventional transaction fee presently is 0.00001 ZEC or 1,000 zatoshis, as
+specified in ZIP 313. This allowed exploration of novel use cases of the Zcash
+blockchain. The Zcash network has operated for almost 2 years at a conventional
+transaction fee of 1,000 zatoshis, without consideration for the total number
+of inputs and outputs in each transaction. Under this conventional fee, some
+usage of the chain has been characterized by high-output transactions with
+1,100 outputs paying the same conventional fee as a transaction with 2 outputs.
+
+The objective of the new fee policy, once it is enforced, is for fees paid by
+transactions to fairly reflect the processing costs that they impose on various
+participants in the network. This will tend to discourage usage patterns that
+cause either intentional or unintentional denial of service, while still
+allowing low fees for regular transaction use cases.
```

Comment:
```suggestion
transactions to fairly reflect the processing costs that their inputs and outputs
impose on various participants in the network. This will tend to discourage
usage patterns that cause either intentional or unintentional denial of service,
while still allowing low fees for regular transaction use cases.
```